workflows4s-quartz/src/test/scala/workflows4s/runtime/wakeup/quartz/QuartzKnockerUpperTest.scala

```diff
@@ -23,11 +24,27 @@ class QuartzKnockerUpperTest extends AnyFreeSpec with Matchers with BeforeAndAft
     "should schedule a wakeup at a specified time" in withQuartzKnockerUpper { knockerUpper =>
       var wokenUpAt: Instant = null
       val testId             = "test-id"
-      knockerUpper.initialize(id => IO { if (id == testId) wokenUpAt = Instant.now() }).unsafeRunSync()
-      val wakeupAt           = Instant.now().plusMillis(100)
+      knockerUpper
+        .initialize(id =>
+          IO {
+            if (id == testId) {
+              wokenUpAt = Instant.now()
+              println(s"Woken up at $wokenUpAt") // Add debug logging
+            }
+          },
+        )
+        .unsafeRunSync()
+      val wakeupAt           = Instant.now().plusMillis(500) // Increased delay to 500ms
```

Comment:
Are you sure this was a problem?  Due to `.unsafeRunSync()` on the line above everything here should be synchronous, meaning it will block until initialization is finished. 

If it's needed, it's fine, it just means the initialization is not handled perfectly - it returns before quartz is fully initialized. But I would like to know if its really needed.